src/Codec/CBOR/Cuddle/CBOR/Validator.hs

```diff
@@ -6,9 +6,9 @@
 
 module Codec.CBOR.Cuddle.CBOR.Validator (
   validateCBOR,
-  validateCBOR',
   CDDLResult (..),
   CBORTermResult (..),
+  ValidatorStage,
```

Comment:
We should also mention in the changelog removal of `validateCBOR'` and exporting of `ValidatorStage`